reporting.md

```diff
@@ -0,0 +1,615 @@
+   discovered, replace the corresponding value extracted with the one found in
+   this record. This permits the Report Receiver to override the report
+   destination. However, to prevent loops or indirect abuse, the overriding URI
+   MUST use the same destination host from the first step.
+
+``` 
+   **TODO FIX to match TLSRPT (instead of DMARC) if we think we need this.**
+
+   For example, if a TLSRPT policy query for "blue.example.com" contained
+   "rua=mailto:reports@red.example.net", the host extracted from the
+   latter ("red.example.net") does not match "blue.example.com", so this
+   procedure is enacted.  A TXT query for
+   "blue.example.com._report._dmarc.red.example.net" is issued.  If a
+   single reply comes back containing a tag of "v=DMARC1", then the
+   relationship between the two is confirmed.  Moreover,
+   "red.example.net" has the opportunity to override the report
+   destination requested by "blue.example.com" if needed.
+
+   Where the above algorithm fails to confirm that the external
+   reporting was authorized by the Report Receiver, the URI MUST be
+   ignored by the Mail Receiver generating the report.  Further, if the
+   confirming record includes a URI whose host is again different than
+   the domain publishing that override, the Mail Receiver generating the
+   report MUST NOT generate a report to either the original or the
+   override URI.
+
+   A Report Receiver publishes such a record in its DNS if it wishes to
+   receive reports for other domains.
+
+   A Report Receiver that is willing to receive reports for any domain
+   can use a wildcard DNS record.  For example, a TXT resource record at
+   "*._report._dmarc.example.com" containing at least "v=DMARC1"
+   confirms that example.com is willing to receive DMARC reports for any
+   domain.
+
+   If the Report Receiver is overcome by volume, it can simply remove
+   the confirming DNS record.  However, due to positive caching, the
+   change could take as long as the time-to-live (TTL) on the record to
+   go into effect.
+
+   A Mail Receiver might decide not to enact this procedure if, for
+   example, it relies on a local list of domains for which external
+   reporting addresses are permitted.
+```
+
+
+# IANA Considerations
+
+There are no IANA considerations at this time.
+
+# Security Considerations
+
+SMTP TLS Reporting provides transparency into misconfigurations and attempts to
+intercept or tamper with mail between hosts who support STARTTLS. There are
+several security risks presented by the existence of this reporting channel:
+
+  * _Flooding of the_ `aggregate-report-uri` _endpoint_: An attacker could
+    flood the endpoint and prevent the receiving domain from accepting
+    additional reports. This type of Denial-of-Service attack would limit
+    visibility into STARTTLS failures, leaving the receiving domain blind to an
+    ongoing attack.
+
+  * _Untrusted content_: An attacker could inject malicious code into the
```

Comment:
What about the risk of using a spoofed TLSRPT record to receive statistics about a domain the attacker does not own?

I think it's worth calling this threat out, especially since we tell senders to disregard STS or DANE policies when delivering reports. The response is, I think, that an attacker who poisons DNS can *today* receive counts of SMTP connections and, generally, most of the same metadata available in the report, so we shouldn't be overly concerned here.